pkg/operator/ceph/controller/spec.go

```diff
@@ -686,8 +685,9 @@ func PodSecurityContext() *v1.SecurityContext {
 // PodSecurityContext detects if the pod needs privileges to run
 func CephSecurityContext() *v1.SecurityContext {
 	context := PodSecurityContext()
-	context.RunAsUser = pointer.Int64(CephUserID)
-	context.RunAsGroup = pointer.Int64(CephUserID)
+	CephUserID := int64(CephUserID)
```

Comment:
instead of updating it here, 
You can define the `CephUserID` as int64 at the top of the file